build.sbt

```diff
@@ -296,7 +296,6 @@ lazy val docs = project
   .settings(
     tlFatalWarnings := false,
     laikaConfig ~= { _.withRawContent },
-    tlSiteApiUrl := Some(url("https://typelevel.org/cats/api/")),
```

Comment:
We now have published unidocs available.
https://s01.oss.sonatype.org/service/local/repositories/releases/archive/org/typelevel/cats-docs_2.13/2.8.0/cats-docs_2.13-2.8.0-javadoc.jar/!/index.html

(javadoc.io is down at the moment)